### PR TITLE
Cleanup constants

### PR DESCRIPTION
**Pull request by jalkoby**
The 2nd part of #311 - cleanup constants (after #312)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,5 @@ pkg
 
 .gems
 .rbenv-gemsets
+
+Gemfile.lock
```
Comment (grosser) [inline .gitignore]: I prefer keeping it to make contribution more stable https://grosser.it/2015/08/14/check-in-your-gemfile-lock/

File: .travis.yml [hunk 2]
```diff
@@ -15,4 +15,6 @@ matrix:
 script: bundle exec rake $TASK
 branches:
   only: master
-before_install: ruby -e "File.write('Gemfile.lock', File.read('Gemfile.lock').split('BUNDLED WITH').first)"
+before_install:
+  - gem uninstall -v '>= 2' -i $(rvm gemdir)@global -ax bundler || true
+  - gem install bundler -v '< 2'
```
Comment (grosser) [inline .travis.yml]: faster / simpler:
`before_install: ruby -e "File.write('Gemfile.lock', File.read('Gemfile.lock').split('BUNDLED WITH').first)"`

File: demo/rails/Gemfile [hunk 3]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
 gem 'dotenv-rails', groups: [:development, :test]
-gem 'rails', '4.2.5'
+gem 'rails', '~> 4.2.11.1'
```
Comment (grosser) [inline demo/rails/Gemfile]: any v4 should do, so this is better/less locked down:
```suggestion
gem 'rails', '~> 4.2'
```

File: recaptcha.gemspec [hunk 4]
```diff
@@ -23,5 +23,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency "pry-byebug"
   s.add_development_dependency "bump"
   s.add_development_dependency "webmock"
-  s.add_development_dependency "rubocop"
+  s.add_development_dependency "rubocop", ">= 0.68.0"
```
Comment (grosser) [inline recaptcha.gemspec]: no need for that if we have a Gemfile.lock :)

File: .travis.yml (hunk 2 above)
Comment (jalkoby) [reply .travis.yml]: actually in #314 I've removed it - all builds works fine

**Pull request merged**